src/frontend/screens/Game/GameSubMenu/index.tsx

```diff
@@ -213,6 +215,8 @@ export default function GamesSubmenu({
     return <CircularProgress className="link button is-text is-link" />
   }
 
+  const showDlcsItem = onShowDlcs && runner === 'legendary'
```

Comment:
maybe we can also check if the game is installed here?

when I open the dialog for a game that is not installed the spinner never goes away and it never shows the install size

![image](https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/assets/188464/b30bc036-381c-4765-8dc4-aba9b29fcde6)

the dlc shows fine in the install dialog

Reply:
ah ok, makes sense. Will fix

Reply:
<img width="475" alt="image" src="https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/assets/26871415/dce7996d-4c04-497f-96d8-f545adeec4eb">